migaloo/assetlist.json

```diff
@@ -143,6 +143,62 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/rac.svg",
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/rac.png"
       }
+    },
+    {
+      "description": "The native stablecoin of Kava",
+      "denom_units": [
+        {
+          "denom": "usdx",
+          "exponent": 0
+        },
+        {
+          "denom": "USDX",
+          "exponent": 6
+        }
+      ],
+      "base": "usdx",
+      "name": "USDX",
+      "display": "USDX",
+      "symbol": "USDX",
```

Comment:
I assume this is ibc-transferred from Kava? In which case can you please register this as an IBC token with a trace from Kava (and kava-migaloo ibc connection must be registered.